src/background.ts

```diff
@@ -120,10 +121,24 @@ const updateInfos = JSON.parse(
   })
 );
 
+// initialize menu
+const menu = MenuBuilder()
+  .setOnLaunchModeItemClicked((useGpu) => {
+    store.set("useGpu", useGpu);
+
+    dialog.showMessageBoxSync(win, {
+      message: "音声合成エンジン起動モードの変更が完了しました",
+      detail: "起動モード変更の反映は次回のソフトウェア起動時に行われます。",
```

Comment:
どの文面にすれば良いのか難しいですが、再起動が必要なことを案内するのが良いと思いました。
```suggestion
      detail: "変更を適用するためにVOICEVOXを再起動してください。",
```

Reply:
確かにユーザが次に何の行動を取れば良いのかわかりやすい文章でいいですね。

エンジンの起動モードを変更しました
変更を適用するためにVOICEVOXを再起動してください。

良さそう